docs/source/indepth_tutorial/directed-search.ipynb

```diff
@@ -98,9 +98,9 @@
      "name": "stderr",
```

Comment:
<blockquote>As can be seen, the merged set of solutions is quite a bit smaller than the sum of solutions across the 5 seeds. This implies that many of the solutions found in each seed land in the same ε-gridcell.  </blockquote><p>This is an awesome explanation, and exactly the things I'm looking for when going through such an example/tutorial!</p><p> </p><p>Maybe, to even further create deep understanding, also process the results with a smaller or larger epsilon value, and show what that does with the (number of) non-dominated solutions.</p><p> </p><blockquote>carefully monitor whether they have converged.</blockquote><p>Converged to what? If you add one line explaining where it's converging to, this section is readable on itself without needing to look that up.</p><p> </p><blockquote>This requires, however, storing </blockquote><p>I think this is a bit clearer:: "The metrics that are collected at runtime, however, need to store"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<ol><li>I don't want to go into different epsilon values. It is considered best practice to use the same epsilon values as used in the optimization. </li><li>Added a few words to this.</li><li>I modified the wording, but in the opposite direction: runtime metrics can be stored afterwards. Post-run calculating metrics requires storing of the archive for every x nfe so that in post, you can calculate the metrics based on these archives. </li></ol>

Reply:
<p>Perfect!</p>